<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>maze</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btu1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 24pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 24pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 24pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btu4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft PhagsPa</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 24pt &quot;Microsoft PhagsPa&quot;;
}</string>
    </property>
    <property name="text">
     <string>制作迷宫</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
